files/scripts/sonic-cfggen

```diff
@@ -0,0 +1,11 @@
+NS="asic" #Network namespace
+[ "$1" -eq "$1" ] 2> /dev/null # check if the first argument is a number
```

Comment:
I see 2 design issues:
1. assume namespace is a number, this may not be true in general.
2. an optional 'DEV' parameter only depending on whether the first one is a number or not. this is not well-understood practice and difficult to extend in future.

Reply:
@qiluo-msft : Added a new script sonic-netns-exec - to execute any command in a given namespace. Removed usage of different wrappers for sonic-db-cli and sonic-cfggen.